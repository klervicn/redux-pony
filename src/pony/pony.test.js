import React from 'react';
import Pony from './pony';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Pony
        key={120}
        id={130}
        name="Poney de test"
        onUpdateClick={jest.fn}
        onDeleteClick={jest.fn}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
