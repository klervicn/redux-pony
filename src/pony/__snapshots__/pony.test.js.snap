// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  style={
    Object {
      "listStyleType": "none",
    }
  }
>
  <input
    onChange={[Function]}
    type="text"
    value="Poney de test"
  />
  <button
    onClick={[Function]}
  >
    Update
  </button>
  <button
    onClick={[Function]}
  >
    Delete
  </button>
</li>
`;
